scio-core/src/main/java/com/google/cloud/dataflow/sdk/io/PatchedAvroIO.java

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
I don't really like how to is diverging from SDK/Beam. We essentially don't want to have this file. Could you create a PR/issue for beam to see if they would accept this?